DOCUMENTATION.md

```diff
@@ -3941,18 +3941,24 @@ balena.models.device.ping('7cf02a6', function(error) {
 <a name="balena.models.device.getStatus"></a>
 
 ##### device.getStatus(device) ⇒ <code>Promise</code>
+Computes the status of an already retrieved device object.
+It's suggested to use `balena.models.device.get(deviceUuid, { $select: ['overall_status'] })` instead.
```

Comment:
```suggestion
We recommend using`balena.models.device.get(deviceUuid, { $select: ['overall_status'] })` instead, as this function will get deprecated in the future.
```

Reply:
@sradevski We decided that we will not deprecate this.
See: https://www.flowdock.com/app/rulemotion/resin-frontend/threads/KlyXZlJ86S-nvOuJbuOdpbmo54i

We are still going to keep it around as a convenience method (which will just return that field) so that the way of retrieving the status is still discoverable by new users. It will still though have the recommendation of using the `device.get()` directly in case they need to retrieve more fields at once.

Reply:
Ah ok then 👍